src/Compilers/CSharp/Test/Symbol/Symbols/Source/NullablePublicAPITests.cs

```diff
@@ -358,7 +358,7 @@ public class C
                 PublicNullableAnnotation.Annotated);
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/35034")]
```

Comment:
>https://github.com/dotnet/roslyn/issues/35034 [](start = 22, length = 45)

Consider capturing the link in WorkItem attribute.

Reply:
I'll address this in a follow up PR.

---
In reply to: [277794882](https://github.com/dotnet/roslyn/pull/35191#discussion_r277794882) [](ancestors = 277794882)